config/application.rb

```diff
@@ -115,6 +115,18 @@ class Application < Rails::Application
     # Path within public/ where assets are compiled to
     config.assets.prefix = "/assets/government-frontend"
 
+    config.govuk_chat_promo_base_paths = %w[
```

Comment:
I think it'd be good to also have a config option here about whether it is enabled that is controlled by a env var. Then we could merge and ship without the env var in place and just need to use that to turn it on and off in production.